GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -420,11 +427,9 @@ async Task LoadLinksForRevisionAsync(GitRevision revision, DistributedSettings s
 
                     string GetLinksForRevision(DistributedSettings settings)
                     {
+                        IEnumerable<ExternalLink> links = _gitRevisionExternalLinksParser.Parse(revision, settings);
                         cancellationToken.ThrowIfCancellationRequested();
```

Comment:
Moved down on purpose because the caller has checked for cancellation right before calling this function.